option optimize_for = LITE_RUNTIME;

message AutoEquipStrengthenRequest
{
 required uint32 posZhengXing = 1;
}
